src/pages/MetricDetail.svelte

```diff
@@ -2,10 +2,126 @@
   import { getMetricData } from "../state/api";
 
   export let params;
+
   const metricDataPromise = getMetricData(params.app, params.metric);
 </script>
 
+<style>
+  .table-header {
```

Comment:
Let's call this "metrics-table" or something, as it's not really a header.

```suggestion
  .metrics-table {
```